log.sh

```diff
@@ -1,16 +1,36 @@
 #!/bin/bash
 
-cd log_files
+# Initialize virtual Python environment.
+
+# module load python3.7.4
+# python3 -m venv py3venv/
+# source py3venv/bin/activate
+# pip install click pandas
+# pip freeze > requirements.txt
```

Comment:
Drop this and document it in the `requirements.txt`?

Reply:
Alternative: Add comment to <https://github.com/ExaESM-WP4/nesh-monitoring/pull/5/files#diff-2e6c5487c5ac414708f45e76fe733399R19> (`# minimal dependencies: Click and pandas`)?